app/src/main/res/values/strings.xml

```diff
@@ -72,8 +72,11 @@
     <string name="delete_pet_image_text">Are you sure you want to delete the profile image of your pet?</string>
     <string name="affirmative_response">Yes</string>
     <string name="negative_response">No</string>
+    <string name="delete_pet">DELETE PET</string>
```

Comment:
It is better if you put instead "Delete pet" and then add the next attribute to whenever you want it to appear all uppercase:
`android:textAllCaps="true"`
```suggestion
    <string name="delete_pet">Delete pet</string>
```